apps/web/components/integrations/CalendarListContainer.tsx

```diff
@@ -217,10 +221,12 @@ export function CalendarListContainer(props: { heading?: boolean; fromOnboarding
       utils.invalidateQueries(["viewer.connectedCalendars"]),
     ]);
   const query = trpc.useQuery(["viewer.connectedCalendars"]);
-  const installedCalendars = trpc.useQuery([
-    "viewer.integrations",
-    { variant: "calendar", onlyInstalled: true },
-  ]);
+  let installedCalendars;
+  if (props.items) {
+    installedCalendars = props.items.filter((item) => item.variant == "conferencing");
+  } else {
+    installedCalendars = trpc.useQuery(["viewer.integrations", { variant: "calendar", onlyInstalled: true }]);
```

Comment:
[Hooks should not be used conditionally](https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level)